apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-index.presenter.tsx

```diff
@@ -1,13 +1,31 @@
 import { IIndexQueryResult, IVocabularyListViewModel } from '@coscrad/api-interfaces';
-import { GenericIndexPresenter } from '../../../utils/generic-components/presenters/generic-index-presenter';
-
-export const VocabularyListIndexPresenter = (
-    indexResult: IIndexQueryResult<IVocabularyListViewModel>
-) => (
-    /**
-     *  TODO [https://www.pivotaltracker.com/story/show/183681839]
-     * We may some day read the actions and allow for bulk command execution in
-     * an index view.
-     */
-    <GenericIndexPresenter {...indexResult} indexLabel={'Vocabulary Lists'} />
-);
+import { HeadingLabel, IndexTable } from '../../../utils/generic-components/presenters/tables';
+import { CellRenderersDefinition } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types/cell-renderers-definition';
+import { renderAggregateIdCell } from '../utils/render-aggregate-id-cell';
+import { formatBilingualText } from './utils';
+
+export const VocabularyListIndexPresenter = ({
+    data: vocabularyListsAndActions,
+}: IIndexQueryResult<IVocabularyListViewModel>) => {
+    const vocabularyLists = vocabularyListsAndActions.map(({ data }) => data);
+
+    const headingLabels: HeadingLabel<IVocabularyListViewModel>[] = [
+        { propertyKey: 'id', headingLabel: 'link' },
```

Comment:
We may want to shift this structure to use dynamic keys sooner rather than later.